plugins/aws/resoto_plugin_aws/resource/cloudwatch.py

```diff
@@ -0,0 +1,127 @@
+        "cloudwatch_state_value": S("StateValue"),
+        "cloudwatch_state_reason": S("StateReason"),
+        "cloudwatch_state_reason_data": S("StateReasonData"),
+        "cloudwatch_state_updated_timestamp": S("StateUpdatedTimestamp"),
+        "cloudwatch_metric_name": S("MetricName"),
+        "cloudwatch_namespace": S("Namespace"),
+        "cloudwatch_statistic": S("Statistic"),
+        "cloudwatch_extended_statistic": S("ExtendedStatistic"),
+        "cloudwatch_dimensions": S("Dimensions", default=[]) >> ForallBend(AwsCloudwatchDimension.mapping),
+        "cloudwatch_period": S("Period"),
+        "cloudwatch_unit": S("Unit"),
+        "cloudwatch_evaluation_periods": S("EvaluationPeriods"),
+        "cloudwatch_datapoints_to_alarm": S("DatapointsToAlarm"),
+        "cloudwatch_threshold": S("Threshold"),
+        "cloudwatch_comparison_operator": S("ComparisonOperator"),
+        "cloudwatch_treat_missing_data": S("TreatMissingData"),
+        "cloudwatch_evaluate_low_sample_count_percentile": S("EvaluateLowSampleCountPercentile"),
+        "cloudwatch_metrics": S("Metrics", default=[]) >> ForallBend(AwsCloudwatchMetricDataQuery.mapping),
+        "cloudwatch_threshold_metric_id": S("ThresholdMetricId"),
+    }
+    cloudwatch_alarm_arn: Optional[str] = field(default=None)
+    cloudwatch_alarm_description: Optional[str] = field(default=None)
+    cloudwatch_actions_enabled: Optional[bool] = field(default=None)
+    cloudwatch_ok_actions: List[str] = field(factory=list)
+    cloudwatch_alarm_actions: List[str] = field(factory=list)
+    cloudwatch_insufficient_data_actions: List[str] = field(factory=list)
+    cloudwatch_state_value: Optional[str] = field(default=None)
+    cloudwatch_state_reason: Optional[str] = field(default=None)
+    cloudwatch_state_reason_data: Optional[str] = field(default=None)
+    cloudwatch_state_updated_timestamp: Optional[datetime] = field(default=None)
+    cloudwatch_metric_name: Optional[str] = field(default=None)
+    cloudwatch_namespace: Optional[str] = field(default=None)
+    cloudwatch_statistic: Optional[str] = field(default=None)
+    cloudwatch_extended_statistic: Optional[str] = field(default=None)
+    cloudwatch_dimensions: List[AwsCloudwatchDimension] = field(factory=list)
+    cloudwatch_period: Optional[int] = field(default=None)
+    cloudwatch_unit: Optional[str] = field(default=None)
+    cloudwatch_evaluation_periods: Optional[int] = field(default=None)
+    cloudwatch_datapoints_to_alarm: Optional[int] = field(default=None)
+    cloudwatch_threshold: Optional[float] = field(default=None)
+    cloudwatch_comparison_operator: Optional[str] = field(default=None)
+    cloudwatch_treat_missing_data: Optional[str] = field(default=None)
+    cloudwatch_evaluate_low_sample_count_percentile: Optional[str] = field(default=None)
+    cloudwatch_metrics: List[AwsCloudwatchMetricDataQuery] = field(factory=list)
+    cloudwatch_threshold_metric_id: Optional[str] = field(default=None)
+
```

Comment:
Please also override the `connect_in_graph` method and create edges to the instances (see accountcollector for reference).